src/components/AddFoodForm.jsx

```diff
@@ -0,0 +1,60 @@
+import {useState} from 'react';
+import { Divider, Input, Button } from 'antd';
+
+
+// Iteration 4
+function AddFoodForm(props) {
+const {createFood} = props
+
+    const [name, setName] = useState('');
+    const [image, setImage] = useState('');
+    const [calories, setCalories] = useState('');
```

Comment:
Here on  
```
const [calories, setCalories] = useState('');
const [servings, setServings] = useState('');
```

You should have it set as 0 because it's a number, like:

```
const [calories, setCalories] = useState(0);
const [servings, setServings] = useState(0);
```